web/client/components/tutorial/Tutorial.jsx

```diff
@@ -167,7 +167,7 @@ class Tutorial extends React.Component {
 
     onTour = (tour) => {
         if (this.props.steps.length > 0 && tour && tour.type) {
-            document.querySelector(tour?.step?.selector)?.scrollIntoView();
+            document.querySelector(tour?.step?.selector)?.scrollIntoView({block: "end"});
```

Comment:
Could we have this value as props so can be changed by cfg in localConfig (if needed)?
By default we could keep `{block: "end"}`.